topdown/http.go

```diff
@@ -272,12 +281,24 @@ func executeHTTPRequest(bctx BuiltinContext, obj ast.Object) (ast.Value, error)
 
 	// format the http result
 	var resultBody interface{}
-	json.NewDecoder(resp.Body).Decode(&resultBody)
+	var resultRawBody []byte
+
+	var buf bytes.Buffer
+	tee := io.TeeReader(resp.Body, &buf)
+	resultRawBody, err = ioutil.ReadAll(tee)
+	if err != nil {
+		return nil, err
+	}
+
+	if isContentTypeJSON(resp.Header) || forceJSONDecode {
+		json.NewDecoder(&buf).Decode(&resultBody)
+	}
 
 	result := make(map[string]interface{})
 	result["status"] = resp.Status
 	result["status_code"] = resp.StatusCode
 	result["body"] = resultBody
+	result["raw_body"] = string(resultRawBody)
```

Comment:
Out of curiosity, what happens if the raw body contains binary characters?

Reply:
If the response body contains binary characters, the raw body will have them too.